digdag-standards/src/main/java/io/digdag/standards/command/DockerCommandExecutor.java

```diff
@@ -80,16 +109,14 @@ private Process startWithDocker(Path projectPath, TaskRequest request, ProcessBu
 
         try {
             // misc
-            command.add("-i");  // enable stdin
             command.add("--rm");  // remove container when exits
 
             // mount
             command.add("-v").add(String.format(ENGLISH,
                         "%s:%s:rw", projectPath, projectPath));  // use projectPath to keep pb.directory() valid
 
-            // workdir
-            Path workdir = (pb.directory() == null) ? Paths.get("") : pb.directory().toPath();
-            command.add("-w").add(workdir.normalize().toAbsolutePath().toString());
+            // working directory
+            command.add("-w").add(projectPath.normalize().toAbsolutePath().toString());
```

Comment:
is using `projectPath` correct? Shouldn't it be `CommandExecutorRequest.getWorkingDirectory()`?

Reply:
Good catch. Fixed it https://github.com/treasure-data/digdag/pull/835/commits/e2e6628a803702a1e450605b4616d30706a6ff31#diff-332f97a55beebab62540b4207e4f8300R116